### ISSUE TITLE
1.6.6 doesn't work at all with firefox 79

### ISSUE THREAD

**Issue by marillat** [keepassxreboot/keepassxc-browser#960]
KeePassXC - 2.6.0
KeePassXC-Browser - 1.6.6
Operating system: Linux Debian unstable amd64
Browser: Firefox 79

I see these errors in Firefox Web console

```
initContentScript error:  SyntaxError: unterminated parenthetical
    matchPatternToRegExp moz-extension://daf3984d-b407-40fe-99df-410adf900c81/global.js:97
    siteMatch moz-extension://daf3984d-b407-40fe-99df-410adf900c81/global.js:101
    siteIgnored moz-extension://daf3984d-b407-40fe-99df-410adf900c81/content/keepassxc-browser.js:1054
    initContentScript moz-extension://daf3984d-b407-40fe-99df-410adf900c81/content/keepassxc-browser.js:889
    async* moz-extension://daf3984d-b407-40fe-99df-410adf900c81/content/keepassxc-browser.js:930
```

**Comment by marillat** [comment]
Works fine with Chromium 83.0.4103.116

**Comment by varjolintu** [comment]
What kind of URL are you using in Site Preferences?

**Comment by marillat** [comment]
for example 
https://www.paypal.com/signin?returnUri=https%3A%2F%2Fwww.paypal.com%2Fmyaccount%2Fsecurity%2F

**Comment by varjolintu** [comment]
Try if using `https://www.paypal.com/*` makes any difference. It should work the same way. So far I haven't been able to reproduce this bug, even using that URL. Can you verify what URL in Site Preferences causes this?

**Comment by marillat** [comment]
Still doesn't work with https://www.paypal.com/* 

I downgraded to 1.6.5 and KeePassXC-Browseris working again.
Error in initContentScript is gone.

**Comment by varjolintu** [comment]
That is strange, because the behaviour of that function hasn't changed. Just the name is. Could you try to update again to 1.6.6 and maybe debug the global.js via Web Inspector and see what the regex is that fails?

**Comment by marillat** [comment]
Well, I'm unlucky. This work now with 1.6.6

**Comment by sylvainfaivre** [comment]
Only adding a comment to note that I had the same problem : KeepassXC browser integration not working, and the same errors in Firefox Web console :
`initContentScript error:  SyntaxError: unterminated parenthetical`
It was also solved by downgrading to 1.6.5, then upgrading again to 1.6.6. I guess it was a matter of uninstalling ang reinstalling the extension ?

**Comment by varjolintu** [comment]
@sylvainfaivre This is a Regex-related error, and I haven't been able to reproduce it. It's even more strange if it's fixed by reinstalling. It's a possible Firefox bug.
